discovery/marathon/marathon.go

```diff
@@ -350,6 +351,12 @@ func targetsForApp(app *App) []model.LabelSet {
 					target[model.LabelName(ln)] = model.LabelValue(lv)
 				}
 			}
+			if i < len(app.Container.PortMappings) {
```

Comment:
Could you add comments indicating this was added to accommodate for the change you noticed with Marthon 1.5.x and another to the previous section indicating that it's for Marthon versions <=  Marathon 1.4.x ?

Reply:
sure - added some comments - let me know if that works!

Reply:
Perfect. I'll merge once the CircleCI tests pass again. Thanks for fixing!